### ISSUE TITLE
DCT - strange mistake, reason unclear

### ISSUE THREAD

**Issue by shariboodts** [ErwinKomen/RU-passim#693]
We've noticed problems in the DCT (case in point: https://passim.rich.ru.nl/dct/details/82/ with Montecassino 12 as pivot manuscript):

First problem:
PS-AU s 249 (item 76 in pivot manuscript) should be visible in DCT as item 70 in Vat. lat. 3836. The AF is properly linked to manifestations in both manuscripts. Why does it not show up in the DCT?
Iris: ![image](https://github.com/ErwinKomen/RU-passim/assets/57531972/2ee602b7-3c9f-488d-bee2-00bcf4e33ca7)

Second problem:
![image](https://github.com/ErwinKomen/RU-passim/assets/57531972/7d1c6960-3603-4b65-9e96-7ff2de8e1a2d)

If this is indeed the case, that is not how it should work, DCT's should always use the most recent, updated version of the manuscripts/AF's the include.

Sometimes when clicking 'View all' on Rows drop-down menu, there is an error: 
![image](https://github.com/ErwinKomen/RU-passim/assets/57531972/7993abef-6615-42ba-aaf2-68b4838d6fa3)
Is this related to the problems described above?

**Comment by Irisdenis** [comment]
Edit: 
Regarding the first problem: this issue seems to be related to the same bug mentioned in issue #745. When first opening the DCT in question,  PS-AU s 249 (nr 76) is _not_ shown, even though it is a match. Only after clicking the option 'Rows = show all', PS-AU s 249 (nr 76) appears as a match in the table. When re-clicking 'Rows = Matches only', the displayed matches seem correct (i.e. nr 76 is displayed). Ergo, the problem is connected to the displaying of matches (not the recognition of matches), just like the examples mentioned in issue #745

**Comment by Irisdenis** [comment]
Another issue, possibly related to problem 2, is **problem 3**:

- In the DCT https://passim.rich.ru.nl/dct/details/23/, with Montpellier H 59 as pivot manuscript, the 'authors' displayed for Montpellier H 59 are not the same ones as the 'attributed authors' defined in the manuscript (e.g. we still find 'Anonymus' as author here, whereas in the manuscript details view, no author is defined. This should be displayed as three dots (...) in the DCT). 
- When deleting and re-adding the manuscript to the DCT, the problem persist.
- When creating a new DCT with this manuscript, the manuscript's authors are displayed correctly.

So: it seems as if old DCT's like this one consistently present the information as it was present in the manuscripts at the time they were added to the DCT (i.e. they are stable, unchanged), _even_ after deleting and re-adding the manuscript to the DCT. **Question**: Is this indeed where the problem originates? 

This results in an incorrect view of the manuscripts in the DCT (i.e. the ms as in the DCT does not correspond to the ms details page). Ideally, a DCT should automatically retrieve 'changed/corrected' data from the manuscripts, or users should at least be able to delete and reload manuscripts that were since corrected (see above).

**Comment by ErwinKomen** [comment]
### Problem 1
This problem was treated in issue #745, and has been resolved. 
For the most part, this was not a problem, but the use of hidden rows by the researcher.
I've added information: when a DCT has hidden rows, then a clear **Warning** is added. And hovering over that warning shows the simple method to remove the hidden rows.

**Comment by ErwinKomen** [comment]
### Problem 2
One large question here is whether the DCT is generated from AF data on a given moment, and not updated, even if that AF is updated.

**Comment by ErwinKomen** [comment]
### Problem 3
Problem:
> Deleting and re-adding the manuscript to a DCT does not update its contents correctly
